JULIERAJ/KIDS-FIRST-Front
# refactored sidebar component

## One Line Description
Split the sidebar component into two components

## Requirements
Split the sidebar component into two components: 1. Sidebar 2. SidebarItemsCard

## Test Steps
1. Open the application.
2. Navigate to the sidebar and verify the sidebar component after the code changes according to the UX/UI design.
## UI/UX Outcome
![image](https://github.com/user-attachments/assets/53c57eb4-1d83-4514-9440-58d1b722b710)
